Debian/bin/lib/Cyrus/Docker/Command/build.pm

```diff
@@ -24,6 +33,29 @@ sub execute ($self, $opt, $args) {
 
   $ENV{CFLAGS}="-g -W -Wall -Wextra -Werror";
 
+  if ($opt->sanitizer) {
+    if ($opt->sanitizer eq 'asan') {
+      $ENV{CYRUS_SAN_FLAGS} = '-fsanitize=address';
+
+      my $lsan_opts = $ENV{LSAN_OPTIONS} || "";
+      my $dont_suppress;
+
+      if ($lsan_opts) {
+        my %opts = map { split '=', $_ } split(':', $lsan_opts);
+        if ($opts{supressions} && $opts{supressions} ne "cunit/leaksanitizer.suppress") {
+          warn "Warning! LSAN_OPTIONS already defines a suppressions file so ours will not be used. You may see spurious failures...\n";
+          $dont_suppress = 1;
+        }
+      }
+
+      unless ($dont_suppress) {
+        $ENV{LSAN_OPTIONS} = "$lsan_opts:suppressions=leaksanitizer.suppress";
+      }
+    } elsif ($opt->sanitizer eq 'ubsan') {
+      $ENV{CYRUS_SAN_FLAGS} = '-fsanitize=undefined';
+    }
```

Comment:
I think you should add `} else { die "unknown --sanitizer option" }`.  It's small, but it'll prevent some amount of future confusion if we add an option and typo a case here, for example

Reply:
done